### ISSUE TITLE
Gamecrash, log spammed with message

### ISSUE THREAD

**Issue by johannesmateboer** [paulevsGitch/BetterNether#402]
After visiting the nether, and playing in the overworld, after a while these messages appear in the log (just a short snippet of a huge list):

Minecraft 1.17.1, latest version BetterNether

```
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=272, y=33, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=272, y=33, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=272, y=33, z=-645}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=271, y=33, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=271, y=33, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=271, y=33, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=271, y=33, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=271, y=33, z=-645}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=270, y=33, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=270, y=33, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=270, y=33, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=270, y=33, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=269, y=33, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=269, y=33, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=269, y=33, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=269, y=33, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=268, y=33, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=268, y=33, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=268, y=33, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=268, y=33, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [16, -41], pos: class_2338{x=267, y=33, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [18, -41], pos: class_2338{x=289, y=34, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [18, -41], pos: class_2338{x=288, y=34, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [18, -41], pos: class_2338{x=288, y=34, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [18, -41], pos: class_2338{x=288, y=34, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [18, -41], pos: class_2338{x=288, y=34, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=287, y=34, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=287, y=34, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=287, y=34, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=287, y=34, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=286, y=34, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=286, y=34, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=286, y=34, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=286, y=34, z=-644}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=285, y=34, z=-641}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=285, y=34, z=-642}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
[09:08:54] [Worker-Main-4/ERROR]: Detected setBlock in a far chunk [17, -41], pos: class_2338{x=285, y=34, z=-643}, status: minecraft:features, currently generating: ResourceKey[minecraft:worldgen/structure_feature / betternether:nether_city]
```

**Comment by paulevsGitch** [comment]
This is a vanilla bug, fix is in dev code

**Comment by quiqueck** [comment]
Fixed in v5.3.5
